services/environment-pruner/prune.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+# set -e -o pipefail
+
+# Create a JWT admin token to talk to the API.
+API_ADMIN_JWT_TOKEN=$(./create_jwt.sh)
+BEARER="Authorization: bearer $API_ADMIN_JWT_TOKEN"
+
+CUTOFF_DATE=$(date +'%Y-%m-%d %H:%M:%S' --date '-90 days')
```

Comment:
should this `-90` be hardcoded, or definable in a variable?

Reply:
Ideally a variable :) the other thing I'm not 100% certain about is if just triggering a delete mutation is enough to remove the environment (it's how we've been doing it) but there might be some oc magic that is needed as well.